src/BenchmarkDotNet/Validators/DotNetSdkVersionValidator.cs

```diff
@@ -113,17 +107,38 @@ private static IEnumerable<string> GetInstalledDotNetSdks(string? customDotNetCl
                     {
                         var output = process.StandardOutput.ReadToEnd();
                         var lines = output.Split(new[] { '\r', '\n' }, StringSplitOptions.RemoveEmptyEntries);
-                        return lines.Select(line => line.Split(' ')[0]); // The SDK version is the first part of each line.
+
+                        var versions = new List<Version>(lines.Count());
+                        foreach (var line in lines)
+                        {
+                            // Each line will start with the SDK version followed by the SDK path. Since we only support
+                            // targeting SDK versions by the major version, we'll just look at extracting the major and
+                            // minor versions. This is done by looking for the first two dots in the line.
+                            var firstDot = line.IndexOf('.');
+                            if (firstDot < 0)
+                                continue;
+
+                            var secondDot = line.IndexOf('.', firstDot + 1);
+                            if (secondDot < 0)
+                                continue;
+
+                            if (Version.TryParse(line.Substring(0, secondDot), out var version))
+                            {
+                                versions.Add(version);
+                            }
+                        }
+
+                        return versions;
```

Comment:
Do need to account for framework versions too? i.e.
```suggestion
                        return lines.Select(line => Version.Parse(line.Split(' ')[0]));
```

Reply:
Framework sdk version are retrieved in a separate method.

Reply:
There are also few others: netcoreapp3.1 and netstandrd2.1 etc. (not sure if tests are exercising them, but we do have tests for those TFMs).

Reply:
This code is comparing major and minor version, so netcoreapp3.1 should work fine.
BDN doesn't support netstandard builds (it must be executable).